app/models/date_time_field_type.rb

```diff
@@ -42,7 +42,7 @@ def mapping_field_name
   end
 
   def timestamp_is_valid?
-    if @timestamp.nil?
+    if @timestamp.blank? || @timestamp.nil?
```

Comment:
A nil-check here is redundant, as the Rails convenience method `blank?` will also return `true` if the object is `nil`.